v2/api/servicebus/customizations/namespace_extensions.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/go-logr/logr"
 	"github.com/pkg/errors"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
minor: weird addition? It's already called `v1`

Reply:
I try deleting it and VSCode adds it back again. Weird. Will tidy using GoLand.

Reply:
Fixed with GoLand.